x/tokenfactory/module.go

```diff
@@ -189,7 +189,7 @@ func (am AppModule) EndBlock(_ sdk.Context, _ abci.RequestEndBlock) []abci.Valid
 // GenerateGenesisState creates a randomized GenState of the tokenfactory module.
 func (am AppModule) SimulatorGenesisState(simState *module.SimulationState, s *simtypes.SimCtx) {
 	tfDefaultGen := types.DefaultGenesis()
-	tfDefaultGen.Params.DenomCreationFee = simulation.TokenFactoryCreationFee
+	tfDefaultGen.Params.DenomCreationFee = sdk.NewCoins(sdk.NewInt64Coin("stake", 10_000_000))
```

Comment:
Can we use `sdk.DefaultBondDenom?

Also, how this `10_000_000` chosen? does it make sense to have it in a constant?

Reply:
Agreed on using sdk.DefaultBondDenom

As for how that number is chosen, I really only want to change the denom to stake, the actual value can be random for all intents and purposes. With that being said because it is an sdk.Coins struct I would need to specify an index of 0 to only change the denom, and that would assume we arent trying to set the fee as a multi coin struct in future tests. What do you think?

Reply:
What if you iterate over all coins in a loop, only changing the denom?

Reply:
If that doesn't work, let's just leave a comment that the value is arbitrary

Reply:
That does work, implementing now!